frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.unit.spec.tsx

```diff
@@ -13,17 +14,17 @@ import {
 } from "./ObjectDetail";
 import type { ObjectDetailProps } from "./types";
 
+const mockQuestion = new Question(
+  createMockCard({
+    name: "Product",
+  }),
+);
+
 function setup(options?: Partial<ObjectDetailProps>) {
```

Comment:
Could you also add basic tests that new menu items are shown when expected? Given that edge cases are tested within the utility, you'd likely need to have 2 tests here - 1 when no new menu items are shown and another one with all menu items